index.js

```diff
@@ -247,7 +248,14 @@ async function getSiteVersion(page) {
 async function launch(url, selector, device) {
     'use strict';
 
-    const browser = await puppeteer.launch();
+    var chrome_args = [];
+    if (config.DISABLE_CHROME_SANDBOX){
+      chrome_args = ['--no-sandbox'];
```

Comment:
```suggestion
      args.push('--no-sandbox');
```

Reply:
Thanks for this, I was not aware of the "push" function to add elements to a const :+1: